scripts/deploy-frontend.sh

```diff
@@ -13,9 +13,10 @@ CF_SPACE=${5}
 ENVIRONMENT=${6}
 
 backend_app_name=$(echo $CGHOSTNAME_BACKEND | cut -d"-" -f3)
```

Comment:
Can we remove this?

Reply:
TODO: Remove this line, backend_app_name is not being used anymore